docs/roman/ramp_fitting/description.rst

```diff
@@ -13,17 +13,21 @@ more detail below.
 
 The count rate for each pixel is determined by a linear fit to the
 cosmic-ray-free and saturation-free ramp intervals for each pixel; hereafter
-this interval will be referred to as a "segment." The fitting algorithm uses an 
+this interval will be referred to as a "segment." The fitting algorithm uses an
 'optimal' weighting scheme, as described by Fixsen et al, PASP, 112, 1350.
 Segments are determined using
 the 3-D GROUPDQ array of the input data set, under the assumption that the jump
 step will have already flagged CR's. Segments are terminated where
-saturation flags are found. Pixels are processed simultaneously in blocks 
+saturation flags are found. Pixels are processed simultaneously in blocks
 using the array-based functionality of numpy.  The size of the block depends
 on the image size and the number of groups.
 
+The ramp fitting step is also where the :ref:`reference pixels <refpix>` are
+trimmed resulting in a smaller array being passed to the subsequent steps.
```

Comment:
suggested  "-" or "," between trimmed and resulting.

Reply:
added